src/uikit.cs

```diff
@@ -26761,5 +27283,1203 @@ interface UITextInputContext {
+
+		[Static]
+		[Export ("requestWithRole:")]
+		UISceneSessionActivationRequest Create (string role);
+
+		[Static]
+		[Export ("requestWithSession:")]
+		UISceneSessionActivationRequest Create (UISceneSession session);
+	}
+
+	[NoWatch, TV (17, 0), iOS (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UISymbolEffectCompletionContext {
+		[Export ("finished")]
+		bool Finished { [Bind ("isFinished")] get; }
+
+		[NullAllowed, Export ("sender", ArgumentSemantic.Weak)]
+		NSObject Sender { get; }
+
+		[NullAllowed, Export ("effect")]
+		NSSymbolEffect Effect { get; }
+
+		[NullAllowed, Export ("contentTransition")]
+		NSSymbolContentTransition ContentTransition { get; }
+	}
+
+	[NoWatch, NoTV, iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UITextItem {
+		[Export ("contentType", ArgumentSemantic.Assign)]
+		UITextItemContentType ContentType { get; }
+
+		[Export ("range", ArgumentSemantic.Assign)]
+		NSRange Range { get; }
+
+		[NullAllowed, Export ("link", ArgumentSemantic.Strong)]
+		NSUrl Link { get; }
+
+		[NullAllowed, Export ("textAttachment", ArgumentSemantic.Strong)]
+		NSTextAttachment TextAttachment { get; }
+
+		[NullAllowed, Export ("tagIdentifier", ArgumentSemantic.Strong)]
+		string TagIdentifier { get; }
+	}
+
+	[NoWatch, NoTV, iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface UIWindowSceneDragInteraction : UIInteraction {
+		[NoMacCatalyst]
+		[Export ("gestureForFailureRelationships")]
+		UIGestureRecognizer GestureForFailureRelationships { get; }
+	}
+
+	[NoWatch, NoTV, iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UITextItemMenuPreview {
+		[Static]
+		[Export ("defaultPreview")]
+		UITextItemMenuPreview DefaultPreview { get; }
+
+		[Export ("initWithView:")]
+		NativeHandle Constructor (UIView view);
+	}
+
+	[NoWatch, NoTV, iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UITextItemMenuConfiguration {
+		[Static]
+		[Export ("configurationWithMenu:")]
+		UITextItemMenuConfiguration Create (UIMenu menu);
+
+		[Static]
+		[Export ("configurationWithPreview:menu:")]
+		UITextItemMenuConfiguration Create ([NullAllowed] UITextItemMenuPreview preview, UIMenu menu);
+	}
+
+	[NoWatch, TV (17, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface UITextLoupeSession {
+		[Static]
+		[Export ("beginLoupeSessionAtPoint:fromSelectionWidgetView:inView:")]
+		[return: NullAllowed]
+		UITextLoupeSession BeginLoupeSession (CGPoint point, [NullAllowed] UIView selectionWidget, UIView interactionView);
+
+		[Export ("moveToPoint:withCaretRect:trackingCaret:")]
+		void MoveToPoint (CGPoint point, CGRect caretRect, bool tracksCaret);
+
+		[Export ("invalidate")]
+		void Invalidate ();
+	}
+
+	interface IUITextCursorView { }
+
+	[NoWatch, TV (17, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[Protocol]
```

Comment:
missing base type/model?

Reply:
Same, this is not going to be implemented by users as opposed to the [UITextSelectionDisplayInteractionDelegate](https://developer.apple.com/documentation/uikit/uitextselectiondisplayinteractiondelegate?language=objc) which does contain Model in it.